library/Core/src/Juvix/Core/Common/Context/RecGroups.hs

```diff
@@ -0,0 +1,32 @@
+-- | Calculate mutually-recursive groups of definitions.
+module Juvix.Core.Common.Context.RecGroups
```

Comment:
Git seems to have got confused about this module moving to `J.C.C.Context.Traverse`. This file (and `…Types`) should probably go